server/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "Backend",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js",
```

Comment:
maybe change start script to `"start": "npx nodemon server.js"`

Reply:
Done.